packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -256,15 +278,33 @@ describe('Model > TutorialStore', function () {
         setTutorialStepSpy.restore()
       }).then(done, done)
     })
+
+    it('should call setSeenTime if the id parameter is defined', function (done) {
```

Comment:
'It should set the seen time if the id is defined'? It would be good if the tests weren't tied to a specific implementation, or specific function names, but just expected certain results.

Reply:
That seems a bit more like integration tests between actions rather than testing each separate action in unit tests? I see your point about not tying the test to a specific function. It's been hard for me to distinguish between unit vs integration with the stores' actions.